packages/ui/src/components/modals/ConditionModal.tsx

```diff
@@ -98,13 +98,13 @@ const constructComparisonTypeDropdownOptions = (options: CLM.ComparisonType[]):
 }
 
 
-const singleEntityDropdownOptions = constructComparisonTypeDropdownOptions(Object.values(CLM.ComparisonType).filter(t => t !== CLM.ComparisonType.NUMBER_OF_ITEMS)) // number of items only valid for multi entities
-const multiEntityDropdownOptions = constructComparisonTypeDropdownOptions(Object.values(CLM.ComparisonType).filter(t => t === CLM.ComparisonType.NUMBER_OF_ITEMS)) // number of items only valid for multi entities
-const enumEntityDropdownOptions = constructComparisonTypeDropdownOptions([])
-var comparisonTypeDropdownOptions = singleEntityDropdownOptions
+const singleEntityDropdownOptions = convertComparisonTypeToDropdownOption(Object.values(CLM.ComparisonType).filter(t => t !== CLM.ComparisonType.NUMBER_OF_VALUES)) // number of items only valid for multi entities
+const multiEntityDropdownOptions = convertComparisonTypeToDropdownOption(Object.values(CLM.ComparisonType).filter(t => t === CLM.ComparisonType.NUMBER_OF_VALUES)) // number of items only valid for multi entities
+const enumEntityDropdownOptions = convertComparisonTypeToDropdownOption(Object.values(CLM.ComparisonType).filter(t => t === CLM.ComparisonType.STRING))
```

Comment:
I'm not sure where my old comment went about why we have these different lists. It seems unnecessary.

What was thoughts on consolidation and available options per comparison type.

Reply:
I thought you said we didn't need these different set of different options anymore since it's always constrained and disabled per my other.

Reply:
No, we still need these, since the set of available options differs depending on the entity type.